lib/ansible/modules/network/ios/ios_interface.py

```diff
@@ -22,6 +22,10 @@
 description:
   - This module provides declarative management of Interfaces
     on Cisco IOS network devices.
+deprecated:
+  removed_in: '2.13'
+  alternative: ios_vlans
```

Comment:
Did you mean to put `ios_interfaces` here?

Also, you need to rename the file with an underscore at the beginning to get rid of the errors.

Reply:
@Qalthos ah thnx for pointing out, moving all modules @ once has resulted in this mess